pysus/online_data/Infodengue.py

```diff
@@ -0,0 +1,100 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import pandas as pd
+from loguru import logger
+
+APP_DIR = Path(__file__).resolve(strict=True).parent.parent
+CID10 = {"dengue": "A90", "chikungunya": "A92.0", "zika": "A928"}
+
+with open(APP_DIR / "dataset/geocode_by_cities.json", "r") as f:
+    geocode_by_cities = json.load(f)
+
+
+def search_cities(city_name: str) -> Dict[str, int]:
```

Comment:
A better argument name here would be `search_string`, since it can be just a few letters